Refresh.GameServer/Types/Matching/RoomStatistics.cs

```diff
@@ -0,0 +1,13 @@
+using Refresh.GameServer.Authentication;
+
+namespace Refresh.GameServer.Types.Matching;
+
+[JsonObject(NamingStrategyType = typeof(CamelCaseNamingStrategy))]
+public class RoomStatistics
+{
+    public ushort PlayerCount { get; set; }
+    public ushort PlayersInPodCount { get; set; }
+    public ushort RoomCount { get; set; }
+    public Dictionary<TokenGame, ushort> PerGame { get; set; }
```

Comment:
These are causing null warnings that shouldn't happen ever, let's just suppress them at the top with `#nullable disable`